### PR TITLE
Combined fixes for minor bugs occurred during the review

### PR DESCRIPTION
**Pull request by kleingeist**

### CODE REVIEW

**Comment by xi** [comment]
The templates look good to me. @2e2a can you check the 629b4a2?

File: apps/budgeting/views.py [hunk 1]
```diff
@@ -90,7 +90,10 @@ class ProposalModerateView(rules_mixins.PermissionRequiredMixin,
     template_name = 'meinberlin_budgeting/proposal_moderate_form.html'
 
     def get_success_url(self):
-        return self.request.path
+        # Multiforms are not working nicely with the UpdateView redirection,
+        # so we have to use internal knowledge of the instantiated multiforms
+        # names for the correct redirection
+        return self.object['feedback'].get_absolute_url()
```
Comment (2e2a) [inline apps/budgeting/views.py]: Could use `self.get_object().get_absolute_url()`

Comment (kleingeist) [inline apps/budgeting/views.py]: hm yes that should work and helps to hide the internal logic in that case. in general i am not happy with multiforms way of rewriting the object

**Comment by 2e2a** [comment]
lgtm

**Pull request merged**